com.unity.render-pipelines.high-definition/Editor/Material/Lit/ShaderGraph/HDLitGUI.cs.meta

```diff
@@ -1,11 +0,0 @@
-fileFormatVersion: 2
-guid: 9932957a675f26c4f80761ce43958e89
```

Comment:
Marker for myself, why don't we remve HDLitGUI.cs ?

Reply:
I think we can